tests/unit/test-class-wc-payments-payment-request.php

```diff
@@ -115,7 +169,7 @@ private static function get_shipping_option( $instance_id ) {
 			'id'     => $method->get_rate_id(),
 			'label'  => $method->title,
 			'detail' => '',
-			'amount' => WC_Stripe_Helper::get_stripe_amount( $method->get_instance_option( 'cost' ) ),
+			'amount' => self::get_stripe_amount( $method->get_instance_option( 'cost' ) ),
```

Comment:
Is there reason not to use the existing `prepare_amount` helper function, which is the equivalent of the existing `get_stripe_amount` helper function used in the original test?

Reply:
Good catch! I remember looking for an alternative function, but I didn't know about `prepare_amount`.

I'll change it now.